app/main.py

```diff
@@ -1 +1,43 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+
+    alive = []
```

Comment:
The `alive` list is currently a class variable, which means it is shared across all instances of `Animal`. This can lead to unintended behavior if multiple instances are created. Consider making `alive` an instance variable to ensure each `Animal` instance maintains its own state.